fix(zoom out): fixed increase zoom out for 3rd option (sphere)

aliraza556 opened the pull request:
### Problem:
When attempting to view the entire graph using the third zoom option from the left, the graph does not fully display. This issue limits the usability of the graph viewing feature, as users are unable to see the complete data representation.

### Solution:
The zoom functionality has been adjusted specifically for the third zoom option. The update ensures that when this option is selected, the graph is scaled appropriately, allowing the entire graph to be viewed without any parts being cut off.

### Changes:
- Modified the zoom scaling factor for the 3rd zoom option in the file `src/components/Universe/Controls/CameraAnimations/index.ts`.
- Adjusted the UI elements to accommodate the change in zoom behavior.
- Enhanced rendering performance for the zoom-out feature to ensure a smooth user experience.
- Attached a video to confirm that 3rd zoom option is handled correctly.

### Testing:
To confirm that the changes effectively resolve the issue, a comprehensive testing process was undertaken. This included:

- Manual testing across different devices and screen resolutions.
- Automated unit tests to verify the new zoom-out functionality.
- Performance testing to ensure no degradation occurs due to the changes.

A video demonstration of the fixed zoom functionality, specifically for the 3rd zoom option, is attached [here](https://www.loom.com/share/941d3cb56b6643fdae67f30072df2e28). This video showcases the zoom-out feature working correctly and displaying the entire graph as intended.

### Evidence Image
![image](https://github.com/stakwork/sphinx-nav-fiber/assets/87068339/a3a34afa-78e4-4c14-954e-c4b2052c6011)


### Notes:
Please review the changes and merge this PR if everything is in order. This fix is essential for the upcoming release, and I look forward to any feedback you might have.

Rassl commented on src/components/Universe/Controls/CameraAnimations/index.ts:
```diff
@@ -81,11 +82,18 @@ export const useCameraAnimations = (
+    if (graphStyle === 'sphere' && cameraControlsRef.current && graphRadius) {
+      cameraControlsRef.current.maxDistance = cameraControlsRef.current.getDistanceToFitSphere(graphRadius + 200)
+      cameraControlsRef.current.maxDistance = 8000;
+      cameraControlsRef.current.minDistance = 200;
```
don't we need to reset this in case if we go to another view?

Rassl commented on src/components/Universe/Controls/CameraAnimations/index.ts:
```diff
@@ -81,11 +82,18 @@ export const useCameraAnimations = (
     }
 
+    if (graphStyle === 'sphere' && cameraControlsRef.current && graphRadius) {
+      cameraControlsRef.current.maxDistance = cameraControlsRef.current.getDistanceToFitSphere(graphRadius + 200)
```
next line will overwrite this, do we need it?

Rassl reviewed (commented):
Please take a look to my comments

aliraza556 commented:
Hi @Rassl 

Resolved Zoom Out Settings
I've addressed the concerns you raised:

1. Removed the redundant 'maxDistance' setting for the 'sphere' graph style. Ensured it only sets once with the correct value.
2. Added logic to reset 'maxDistance' and 'minDistance' when switching to a different view or graph style.

These changes should ensure more consistent behavior of the camera controls across different views and styles.

Rassl reviewed (approved):
LGTM
